ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -290,6 +290,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getOwnerStatus = (ownerStatus: string): string => {
+      if (ownerStatus) {
+       if(ownerStatus=='PREVIOUS')
```

Comment:
Minor spacing issues (linter might complain) 
also the equality operator we use traditionally  is the strict equality (`===`), which also checks for type. 
The `==` equality operator will type coerce properties. 
Super small thing but good for consistency 👍 

`if (ownerStatus === 'PREVIOUS')`

Reply:
Good point.